src/scrapers/facebook.iced

```diff
@@ -108,12 +110,3 @@ exports.FacebookScraper = class FacebookScraper extends BaseScraper
       if proof_text_check.split(/\s+/).indexOf(med_id)  < 0
```

Comment:
Actually.. Can you make this stronger by requiring the format `Verifying myself: I am (\\S+) on Keybase.io. (\\S+)` like in the go version? And assert the username.